FlowCrypt/Functionality/Services/Local Private Key Services/InMemoryPassPhraseStorage.swift

```diff
@@ -57,12 +57,14 @@ final class InMemoryPassPhraseStorage: PassPhraseStorageType {
 
                 return isPassPhraseValid ? passPhrase : nil
             }
+            .filter { $0.email == email }
     }
 }
 
 extension InMemoryPassPhraseStorage: LogOutHandler {
     func logOutUser(email: String) throws {
```

Comment:
should be called `removePassPhrases` and doesnt have to be LogOutHandler. in another issue